cmd/juju/interact/pollster_test.go

```diff
@@ -184,6 +184,78 @@ func (PollsterSuite) TestEnterEmpty(c *gc.C) {
 	c.Assert(squash(w.String()), jc.Contains, "Enter your name: Enter your name: ")
 }
 
+func (PollsterSuite) TestEnterVerify(c *gc.C) {
```

Comment:
I've been complaining in recent reviews that test names should not just state what they're testing, but the expected outcome.